Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -538,19 +548,18 @@ def puts_results(self, learners, results, class_var):
 
     def _update_header(self):
         # Set the correct horizontal header labels on the results_model.
-        headers = ["Method"]
         if self.data is not None:
-            if self.data.domain.has_discrete_class:
-                headers.extend(classification_stats.headers)
-            else:
-                headers.extend(regression_stats.headers)
-
-        # remove possible extra columns from the model.
-        for i in reversed(range(len(headers),
-                                self.result_model.columnCount())):
-            self.result_model.takeColumn(i)
-
-        self.result_model.setHorizontalHeaderLabels(headers)
+            func = classification_stats if self.data.domain.has_discrete_class \
+                else regression_stats
+            scores = func.scores
+        else:
+            scores = []
+        model = self.result_model
+        model.setColumnCount(1 + len(scores))
+        for col_1, score in enumerate(scores):
```

Comment:
It stood for `col - 1`. Apparently only I use this convention, so I removed `_1`.